understanding/20/name-role-value.html

```diff
@@ -14,18 +14,18 @@ <h2>Intent of Name, Role, Value</h2>
       
       
       <p>The intent of this Success Criterion is to ensure that Assistive Technologies (AT)
-         can gather information about, activate(or set) and keep up to date on the status of
+         can gather accurate information about, activate (or set) and keep up to date on the status of
```

Comment:
suggest changing `accurate` to `appropriate`, as even "accurate" implies there's always an unambiguously correct/proper/accurate one